README.md

```diff
@@ -149,9 +149,9 @@ These methods are available on a Writer object:
   Connect to the nsqd specified.
 * `close()` <br/>
   Disconnect from nsqd.
-* `publish(topic, msgs, [callback])` <br/>
+* `publish(topic, msgs, [timeMs], [callback])` <br/>
```

Comment:
I think we should strive for API clarity. I'm not sure `_publish` is the right way to go. Rather, it might more sense to refactor the checks needed to DRY it up.